cisstMultiTask/mtsCommandVoidReturn.h

```diff
@@ -91,7 +91,7 @@ class CISST_EXPORT mtsCommandVoidReturn: public mtsCommandBase {
       This is intended for derived classes (e.g., mtsCommandQueuedVoidReturn). */
     virtual mtsExecutionResult Execute(mtsGenericObject & result,
                                        mtsCommandWriteBase * CMN_UNUSED(finishedEventHandler))
-    { return Execute(result, 0); }
+    { return Execute(result); }
```

Comment:
Is this the right fix here? It definitely fixes the problematic infinite recursion, but is it delegating to the correct method...?

Reply:
Yes, that is the right fix.